src/System.Net.Requests/src/System/Net/WebExceptionPal.Unix.cs

```diff
@@ -11,26 +11,14 @@ public partial class WebException : InvalidOperationException
     {
         internal static WebExceptionStatus GetStatusFromException(HttpRequestException ex)
         {
-            WebExceptionStatus status;
-
             // Issue 2384: update WebException.GetStatusFromException after System.Net.Http API changes
             //
             // For now, we use the .HResult of the exception to help us map to a suitable
             // WebExceptionStatus enum value.  The .HResult is set into this exception by
             // the underlying .NET Core and .NET Native versions of the System.Net.Http stack.
             // In the future, the HttpRequestException will have its own .Status property that is
             // an enum type that is more compatible directly with the WebExceptionStatus enum.
-            switch (ex.HResult)
-            {
-                case (int)Interop.Http.CURLcode.CURLE_COULDNT_RESOLVE_HOST:
-                    status = WebExceptionStatus.NameResolutionFailure;
-                    break;
-                default:
-                    status = GetStatusFromExceptionHelper(ex);
-                    break;
-            }
-
-            return status;
+            return GetStatusFromExceptionHelper(ex);
```

Comment:
Since this method no longer does anything useful, can we remove it and rename `GetStatusFromExceptionHelper` to `GetStatusFromException`?

Reply:
No, there's a different implementation for Win, also internally using `GetStatusFromExceptionHelper`, see [WebExceptionPal.Windows.cs](https://github.com/dotnet/corefx/blob/7a318879f787e9feee9266fe9e876ee2411b548f/src/System.Net.Requests/src/System/Net/WebExceptionPal.Windows.cs#L21)